src/cases/containers/cases-create/cases-create.component.ts

```diff
@@ -50,4 +107,22 @@ export class CasesCreateComponent {
     this.logger.info(this.caseType);
   }
 
+  // for search results
+
+  ngOnInit() {
+    this.jurisdictions = [this.JURISDICTION_1, this.JURISDICTION_2];
+  }
+
+  applied(selected) {
+    console.log('selected:', selected);
```

Comment:
applied, reset and jusristictionSelected will get picked up by connector component

Instead what we should be providing is biding array that binds those events to exui actions to connector component

same as on case create